tests/test_hubspot_automatic_fields.py

```diff
@@ -69,7 +69,7 @@ def test_run(self):
                 expected_keys = self.expected_automatic_fields().get(stream)
 
                 # BUG_TDL-9939 https://jira.talendforge.org/browse/TDL-9939 Replication keys are not included as an automatic field for these streams
-                if stream in {'companies', 'deals', 'subscription_changes', 'email_events'}:
+                if stream in {'companies', 'subscription_changes', 'email_events'}:
```

Comment:
Can we make this change for the other streams with replication keys that are missing automatic inclusion?

Reply:
@kspeer825 Yes, the replication key is not automatic for companies, subscription_changes, and email_events streams.

1. The `companies` stream has the same issue([link](https://talend.slack.com/archives/C01TGC4QAUE/p1653902238364979?thread_ts=1653593162.728269&cid=C01TGC4QAUE)) of the replication key as the `deals` stream. So, can we create a card to resolve this issue and work on that?
2.  `subscription_changes` and `email_events` streams are retrieving records in the date window. These streams are saving the `start date of the last window` as a bookmark value in `startTimestamp` field.
`email_events` stream has a `created` field in the API response and `subscription_changes` has a `timestamp` field in the response that we may use as a replication key for the respective stream. But, we are not sure whether filter parameters(`Start timestamp`, `End timestamp`) are following these keys or not as we can't find any doc stating the same, so we asked a question for the same in the community([link](https://community.hubspot.com/t5/APIs-Integrations/Clarity-on-parameter-of-API/td-p/657281)).
  **Possible solutions:** 
    1. We can add a new `startTimestamp` field in the record explicitly and mark that field as an automatic to keep current behavior as is.
    2. Update the bookmark key for `email_events` and `subscription_changes` as `created` and `timestamp` respectively if we got a positive reply from the support.

Reply:
1. Sounds good 
2. Are you able to verify the usage of these keys by exploratory testing? If you have a record before the StartTimestamp, another record after the EndTimestamp, and another record between those two I think you'd answer that question. I supposed you'd actually need two more records at either of those fence posts to see if the API treats either inclusively.

Reply:
As per discussion with @cosimon, we concluded to make a change in the replication key just for the deals stream. We are not making any changes in `subscription_changes` and `email_events` as of now.